views/components/ConfigurationsList/index.jsx

```diff
@@ -484,10 +485,12 @@ const ConfigurationsList = ({ user, classes, theme, navigate }) => {
           </label>
         </form>
         <Button
+          component={Link}
           variant="fab"
           color="secondary"
-          href={routes.createConfiguration}
+          to={routes.newConfiguration}
           focusRipple
+          fab
```

Comment:
Where do the `to` and `fab` props get implemented? Does the `component` prop allow us to pass `Link` props to this component?

Reply:
Yes the component prop allows us to pass the Link props to the component. The to is for routing and the fab belongs to the material ui component.